### PR TITLE
Remove ESP8266 support, clean up repository

### PR DESCRIPTION
**Pull request by fiendie**

### CODE REVIEW

File: src/Displaytemplatestandard.h [hunk 1]
```diff
@@ -65,15 +68,6 @@ void printScreen()
         //draw setPoint line
         u8g2.drawLine(18, 58 - (setPoint / 2), 23, 58 - (setPoint / 2));
 
-        // PID values over heat bar, TOF if active
```
Comment (kjyv) [inline src/Displaytemplatestandard.h:68]: only the TOF part of the comment should go, the other part is still correct

File: src/EmbeddedWebserver.h [hunk 2]
```diff
@@ -68,8 +58,8 @@ int historyValueCount = 0;
 void serverSetup();
 void setEepromWriteFcn(int (*fcnPtr)(void));
 
-// We define these in rancilio-pid.cpp
-#define EDITABLE_VARS_LEN 26
+// Defined in main.cpp
```
Comment (kjyv) [inline src/EmbeddedWebserver.h:61]:
```suggestion
// editable vars are specified in main.cpp
```

File: src/Displaytemplatestandard.h [hunk 3]
```diff
@@ -65,15 +68,6 @@ void printScreen()
         //draw setPoint line
         u8g2.drawLine(18, 58 - (setPoint / 2), 23, 58 - (setPoint / 2));
```
Comment (kjyv) [inline src/Displaytemplatestandard.h:70]: // PID values over heat bar

Comment (kjyv) [inline src/Displaytemplatestandard.h:70]:
```suggestion
// PID values over heat bar
```

File: src/Displaytemplatestandard.h [hunk 4]
```diff
@@ -64,16 +67,7 @@ void printScreen()
...
-        // PID values over heat bar, TOF if active
-    #if TOF == 1
-        if (percentage < 10.00) {
-            if (isrCounter < 500) {
-                u8g2.setCursor(40, 48);
-                u8g2.print(langstring_waterempty);
-            }
-        }
-    #else
+// PID values over heat bar
```
Comment (kjyv) [inline src/Displaytemplatestandard.h:70]:
```suggestion
    // PID values over heat bar
```

Comment (kjyv) [reply src/Displaytemplatestandard.h:70]: sorry ;)

Comment (fiendie) [reply src/Displaytemplatestandard.h:70]: We should think about cleaning the comments and indentation in one go :)

**Pull request merged**